bigbluebutton-html5/imports/ui/components/chat/message-form/container.jsx

```diff
@@ -25,7 +26,7 @@ export default withTracker(() => {
   const stopUserTyping = () => makeCall('stopUserTyping');
 
   return {
-    startUserTyping,
+    startUserTyping: _.throttle(startUserTyping, 2000),
```

Comment:
Wouldn't it make sense to make the throttle wait time configurable like it was done with cursor and annotation?

Reply:
IMO this is a value that we'll don't need to tweak, also the start/stop typing feature uses another wait time in server. If we leave this one to be configurable, we'll need the other value to be configurable [too](https://github.com/bigbluebutton/bigbluebutton/blob/develop/bigbluebutton-html5/imports/api/group-chat-msg/server/modifiers/startTyping.js#L7), and maybe some  misconfiguration could cause a unexpected behavior of this feature.